lib/season.rb

```diff
@@ -36,4 +36,23 @@ def self.generate_seasons(games)
     end
     seasons_hash
   end
+
+  def shots_by_team_per_season_avg #helper method for most and least accurate
+    shots_by_teams = Hash.new(1)
+
+    @games_in_season.each do |game|
+      shots_by_teams[game.teams_game_stats[:home_team][:team_name]] *= game.teams_game_stats[:home_team][:shots].to_f / game.teams_game_stats[:home_team][:goals]
```

Comment:
this line seams a little long. maybe on a future refactor you could assign variables to different pieces of this to shorten it up?